operations-stdlib/src/commonTest/kotlin/string/CompareToDateTest.kt

```diff
@@ -0,0 +1,178 @@
+package string
+
+import JsonLogicResult.Success
+import JsonLogicResult.Failure
+import JsonLogicEngine
+import TestInput
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import string.compareToDate.CompareToDate
+
+class CompareToDateTest:FunSpec({
```

Comment:
incorrect formatting (pls run autoformatting (option + cmd + L in InteliJ) in this file and in `CompareToDate`)